src/widgets/splits/SplitContainer.cpp

```diff
@@ -643,14 +643,7 @@ void SplitContainer::paintEvent(QPaintEvent * /*event*/)
     }
     else
     {
-        if (getApp()->getThemes()->isLightTheme())
-        {
-            painter.fillRect(rect(), QColor("#999"));
-        }
-        else
-        {
-            painter.fillRect(rect(), QColor("#555"));
-        }
+        painter.fillRect(rect(), this->theme->splits.background);
```

Comment:
If I remember correctly, this is the color that's shown between splits. At least in the dark theme, it's the same as `tabs.dividerLine` (not in light themes, though).